src/tiered/tiered_handle.c

```diff
@@ -129,6 +129,18 @@ __tiered_init_tiers(WT_SESSION_IMPL *session, WT_TIERED *tiered, WT_CONFIG_ITEM
     WT_DECL_ITEM(tmp);
     WT_DECL_RET;
     WT_TIERED_TIERS *local_tier;
+    uint32_t object_id;
+    const char *cfg, *name;
+
+    if (session->import_list != NULL) {
+        /* Insert metadata for all tiered objects. */
+        for (object_id = 1; object_id < tiered->current_id; object_id++) {
```

Comment:
The `1` here should really be `tiered->oldest_id` so that it works when we implement garbage collection. For now that field will always be 1.

Reply:
Yeah, I wasn't confident to hardcode `1` here. Thank you for the suggestion, it's much better now!